dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java

```diff
@@ -584,7 +581,15 @@ else if (_gotWriter == true) {
 // code above LIBERTY only
             }
             catch (Throwable th) {
-                logger.logp(Level.SEVERE, CLASS_NAME,"closeResponseOutput", "Error.while.closing.response.output",th);
+                //Client can close the connection and cause this closing error. Since there is no re-throw backup and the response is winding down,
+                //there is no beneficial other than this random alarm message(s) in console log.
```

Comment:
there is no benefit other than this random alarming message(s) in the console log.